uxarray/grid/utils.py

```diff
@@ -423,3 +428,168 @@ def _get_lonlat_rad_face_edge_nodes(
     face_edges_lonlat_rad[valid_mask, 1] = node_lat_rad[valid_edges]
 
     return face_edges_lonlat_rad.reshape(n_face, n_max_face_edges, 2, 2)
+
+
+def _xyz_to_lonlat_rad(
```

Comment:
Wondering why these were moved from the `coordinates` module?

Reply:
cyclical dependencies issue